src/py4vasp/_raw/definition.py

```diff
@@ -489,6 +505,14 @@ def selections(quantity):
     cell=Link("cell", "final"),
     positions="results/positions/position_ions",
 )
+schema.add(
+    raw.Structure,
+    name="exciton",
+    required=raw.Version(6, 5),
+    cell=Link("cell", "exciton"),
+    topology=Link("topology", "exciton"),
+    positions="results/supercell/position_ions",
+)
```

Comment:
Is this supercell structure only used by excitons or could it be possibly be used by other parts? If you think that is would be reusable, you could give it the name "supercell"; similar for cell and topology.